juju/juju
# Unwind jujud

The following starts to unwind jujud from the commands inside of the cmd package.
The idea is that jujud should depend on other things, not other things should depend
on jujud. There is quite a lot of work involved here, but it should mean that in the future
we don't get impacted by the dqlite causing other binaries requiring dqlite for building.

There is a lot more to do, but this is phase 1.

## Checklist


## QA steps

```sh
$ juju bootstrap lxd test --build-agent
$ juju enable-ha
$ juju add-model default
$ juju deploy ubuntu
```

```sh
$ juju bootstrap microk8s test
$ juju add-model dummy
$ juju deploy postgresql-k8s --channel 14/stable
```

All the tests should pass.
